### PR TITLE
Feature database seeding script and unit tests review2

### PR DESCRIPTION
**Pull request by amolgajewar**

### CODE REVIEW

**Comment by tf3** [comment]
My main concern is with how you're handling the image url. The way it works now, the user will see a random image each time they load the same product. And on my understanding of the requirements, that's not how it's supposed to work. On my understanding, the user should see the same data each time they load the same product.

Of course, that's just my understanding, and if you want to you can go ahead and merge.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "peoplealsoviewed",
+  "version": "1.0.0",
+  "description": "People who viewed this item also viewed",
+  "main": ".eslintrc.js",
```
Comment (therobinkim) [inline package.json]: `.eslintrc.js` is not an appropriate `main` file.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "url": "git+https://github.com/fec4-gandolf/PeopleAlsoViewed.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/fec4-gandolf/PeopleAlsoViewed/issues"
+  },
+  "homepage": "https://github.com/fec4-gandolf/PeopleAlsoViewed#readme",
+  "dependencies": {
+    "Faker": "^0.7.2",
```
Comment (therobinkim) [inline package.json]: Don't use `Faker`: https://www.npmjs.com/package/Faker

File: seedAlsoViewed.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  return 'https://picsum.photos/200/300';
+};
+
+/// randomly returns free shipping true or false
+var getFreeShipping = function () {
+  return (Math.round(Math.random() * 10) > 5);
+};
+
+/// based on free shipping true or false returns random shipping cost under 100
+var getShippingCost = function (freeSheeping) {
```
Comment (therobinkim) [inline seedAlsoViewed.js]: `freeSheeping` -> `freeShipping`

File: seedAlsoViewed.test.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    done();
+  };
+
+  var queryString = 'select * from alsovieweditems where id = 1';
+  var queryArgs = [];
+  dbConnection.query(queryString, queryArgs, callback);
+});
+
+test('read query on alsovieweditems should return object with colomn names as properties', done => {
+  var callback = function (err, results) {
```
Comment (therobinkim) [inline seedAlsoViewed.test.js]: Creating your callback first for such a simple operation seems unnecessary. I would recommend creating an inline function on line 67 instead.

**Comment by amolgajewar** [comment]
closing pull request to implement reviews.

**Comment by therobinkim** [comment]
Just leave the PR open, push up your changes, and get it re-reviewed when you're ready.

**Review by NapoleonDLP — COMMENTED** [review]
Would README be a better place for the instructions on creating the DATABASE?

**Review by mmcphillips — APPROVED** [review]
The randomly discount token looks like a great way to mock real e-commerce data.
I believe my only suggestion would be to look at the size of your image string. Although it shouldn't matter for a project of this scale, having an excess of empty characters when allocating memory for sql entries could waste space in larger projects

**Comment by amolgajewar** [comment]
> The randomly discount token looks like a great way to mock real e-commerce data.
> I believe my only suggestion would be to look at the size of your image string. Although it shouldn't matter for a project of this scale, having an excess of empty characters when allocating memory for sql entries could waste space in larger projects

Sure. I have generalized it for now but can be reduced.

**Comment by amolgajewar** [comment]
> Would README be a better place for the instructions on creating the DATABASE?

Yes. Need to add instructions for seeding and testing in README.

**Pull request merged**